utils/ExampleRunner/example-info.json

```diff
@@ -274,6 +274,10 @@
       "generateImageFromTimeData": {
         "name": "Generate 3D Volume From 4D Data",
         "description": "Demostrates generating a 3D volume from 4D data using subtract, average or sum."
+      },
+      "mprVolume3d": {
```

Comment:
what is the difference between this example and the ptct example we have here? https://www.cornerstonejs.org/live-examples/petct

Reply:
Hi @sedghi, it's exactly the same example but Volume3D instead of Volume in the MIP viewport, now I've just modified the PTCT example by adding a button to switch to Volume3D.

Reply:
That would be much better thanks